### DISCUSSION TITLE
Browser extension or Electron?（浏览器插件还是 Electron 客户端？）

### DISCUSSION THREAD

**Discussion by Xheldon** [Xheldon/Notion-Flow-Prod#1] [Polls]
Which do you prefer for the functionality of this product, a browser extension implementation or a client implementation built with Electron?（对于本产品的功能，你是更喜欢浏览器插件实现，还是 Electron 构建的客户端实现呢？）
---

**Comment by msdone-lwt** [comment]
小白问一句，图片能不能不用 oss、cdn 啥的，直接用 notion 里面图片的直链

**Reply by Xheldon** [reply]
Notion 的图片、文件链接有过期时间的，这个是 Notion 的限制，要不都把 Notion 当图床或者网盘，它不得破产了😂![image](https://github.com/Xheldon/Notion-Flow-Prod/assets/9780154/a79bd685-9ee5-4cf5-b8ba-556adbbe0b94)

**Reply by Xheldon** [reply]
不过 Notion Next 这个产品是这么做的，对小白更友好，它貌似会在图片过期前自动重新请求一次 Notion 图片地址来更新博客中的图片，毕竟它是需要服务器的产品，可以做到这一点，Notion Flow 只能手动点部署哈

**Reply by msdone-lwt** [reply]
谢谢 了解了
